gradlew

```diff
@@ -0,0 +1,244 @@
+#!/bin/sh
```

Comment:
As far as I know it's a file generated by gradle. Do we have to put that in git? Same question for `gradlew.bat` and `gradle-wrapper` files.

Reply:
So these files are automatically generated by gradle.
We can add these under gitignore, but it would be required to re run the gradle config and download them, to run the project.

Reply:
What's the best practice? Add them to .gitignore to keep the repo clean or just checkout these files?

Reply:
This is what i have seen on the web.
It is better to add them into the VCS

Reason
Standardizes a project on a given Gradle version, leading to more reliable and robust builds.

Provisioning a new Gradle version to different users and execution environment (e.g. IDEs or Continuous Integration servers) is as simple as changing the Wrapper definition.

Reply:
Ok sounds good